angular/core/components/proposal_redirect/proposal_redirect_controller.coffee

```diff
@@ -1,11 +1,12 @@
-angular.module('loomioApp').controller 'ProposalRedirectController', ($router, $rootScope, $routeParams, $location, Records, LmoUrlService) ->
+angular.module('loomioApp').controller 'ProposalRedirectController', ($router, $timeout, $rootScope, $routeParams, $location, Records, LmoUrlService) ->
   $rootScope.$broadcast('currentComponent', 'proposalRedirect')
   Records.proposals.findOrFetchById($routeParams.key).then (proposal) =>
     Records.discussions.findOrFetchById(proposal.discussionId).then (discussion) =>
-      params = {proposal: proposal.key}
+      params = {}
       if $location.search().position?
         params.position = $location.search().position
-      $location.url LmoUrlService.discussion discussion, params
+        $location.search 'position', null
+      $timeout -> $location.path LmoUrlService.proposal proposal, params
```

Comment:
This broke and I don't know why. Pulling it out of the digest loop fixed it, tho